packages/calico/extra/dcos-calico-node.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=DC/OS calico node: initializes a new calico node and runs calico components
+After=docker.service
+Requires=docker.service
+ConditionPathExists=/opt/mesosphere/etc/calico_enabled
```

Comment:
Do we really need this? How about checking Calico enable in `dcos-calico-node.sh` script?

Reply:
It's necessary to keep it.
1. This is a common usage for systemd to disable a systemd unit, also, this condition check is used by other unit, like [`dcos-docker-gc`](https://github.com/dcos/dcos/blob/master/packages/docker-gc/extra/dcos-docker-gc.service#L3)
2. all the systemd units which are not disabled should be inactive state, except those are set as exceptions like in [cloud-config.yaml](https://github.com/dcos/dcos/blob/master/gen/coreos/cloud-config.yaml).